### DISCUSSION TITLE
Favorite pattern for loading related objects?

### DISCUSSION THREAD

**Discussion by ProTip** [mobxjs/mobx#3746] [General]
I feel like I'm implementing a different pattern ad-hoc on every project for loading an aggregate across separate API endpoints.

Linear appears to have created a sort of "reverse ORM" via decorators that allows them to fixup relationships as objects and updates come in.

How are others handling this on larger projects in a clean, efficient manner?

**Comment by mweststrate** [comment]
See also serializr and mobx-state-tree packages
